node/tests/redis-modules/RedisModules.test.ts

```diff
@@ -0,0 +1,348 @@
+import { afterAll, beforeAll, describe, expect, it } from "@jest/globals";
+import { v4 as uuidv4 } from "uuid";
+import {
+    BaseClientConfiguration,
+    ConditionalChange,
+    InfoOptions,
+    ProtocolVersion,
+    RedisClient,
+    RedisClusterClient,
+    parseInfoResponse,
+} from "../../";
+import * as json from "../../src/redis-modules/Json";
+import { RedisCluster, getFirstResult } from "../TestUtilities";
+
+type Context = {
+    client: RedisClusterClient | RedisClient;
+};
+
+type BaseClient = RedisClusterClient | RedisClient;
+
+const TIMEOUT = 20000;
+
+describe("RedisModules", () => {
```

Comment:
Since we have the redis-modules folder, can we seperate test files by module type? e.g. JsonModule.test.ts